src/modules/create-market/components/create-market-review.jsx

```diff
@@ -3,7 +3,7 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { augur } from 'services/augurjs'
-import BigNumber from 'bignumber.js'
+import BigNumberLib from 'bignumber.js'
```

Comment:
Suggest using `import BigNumber from 'bignumber.js'` to keep the naming consistent.